tests/integration/data/tostac/ga_ls8c_ard_3-1-0_088080_2020-05-25_final.stac-item_expected.json

```diff
@@ -1,1080 +1,1128 @@
-        },
-        {
-            "title": "ODC Dataset YAML",
-            "rel": "odc_yaml",
-            "type": "text/yaml",
-            "href": "http://dea-public-data-dev.s3-ap-southeast-2.amazonaws.com/analysis-ready-data/ga_ls8c_ard_3/088/080/2020/05/25/ga_ls8c_ard_3-1-0_088080_2020-05-25_final.odc-metadata.yaml"
-        },
-        {
-            "rel": "collection",
-            "href": "https://explorer.dev.dea.ga.gov.au/stac/collections/ga_ls8c_ard_3"
-        },
-        {
-            "title": "ODC Product Overview",
-            "rel": "product_overview",
-            "type": "text/html",
-            "href": "https://explorer.dev.dea.ga.gov.au/product/ga_ls8c_ard_3"
-        },
-        {
-            "title": "ODC Dataset Overview",
-            "rel": "alternative",
-            "type": "text/html",
-            "href": "https://explorer.dev.dea.ga.gov.au/dataset/2aa69fcf-aa55-4747-9d95-3652f9fe79b0"
-        }
-    ]
-}
+  "type": "Feature",
+  "stac_version": "1.0.0",
+  "id": "2aa69fcf-aa55-4747-9d95-3652f9fe79b0",
+  "properties": {
+      "title": "ga_ls8c_ard_3-1-0_088080_2020-05-25_final",
+      "dea:dataset_maturity": "final",
+      "end_datetime": "2020-05-25T23:36:02.557719Z",
+      "start_datetime": "2020-05-25T23:35:32.815426Z",
+      "eo:cloud_cover": 42.42102985910952,
+      "gsd": 15.0,
+      "instruments": [
+          "oli",
+          "tirs"
+      ],
+      "platform": "landsat-8",
+      "view:sun_azimuth": 34.22994171,
+      "view:sun_elevation": 31.7895917,
+      "fmask:clear": 1.0427704509544131,
+      "fmask:cloud": 42.42102985910952,
+      "fmask:cloud_shadow": 2.919727915379625,
+      "fmask:snow": 0.00034972190505407335,
+      "fmask:water": 53.61612205265139,
+      "gqa:abs_iterative_mean_x": 0.17,
+      "gqa:abs_iterative_mean_xy": 0.24,
+      "gqa:abs_iterative_mean_y": 0.16,
+      "gqa:abs_x": 0.28,
+      "gqa:abs_xy": 0.4,
+      "gqa:abs_y": 0.28,
+      "gqa:cep90": 0.4,
+      "gqa:iterative_mean_x": 0.03,
+      "gqa:iterative_mean_xy": 0.12,
+      "gqa:iterative_mean_y": 0.12,
+      "gqa:iterative_stddev_x": 0.21,
+      "gqa:iterative_stddev_xy": 0.26,
+      "gqa:iterative_stddev_y": 0.15,
+      "gqa:mean_x": -0.08,
+      "gqa:mean_xy": 0.15,
+      "gqa:mean_y": 0.12,
+      "gqa:stddev_x": 0.42,
+      "gqa:stddev_xy": 0.58,
+      "gqa:stddev_y": 0.4,
+      "landsat:collection_category": "T1",
+      "landsat:collection_number": 1,
+      "landsat:landsat_product_id": "LC08_L1TP_088080_20200525_20200608_01_T1",
+      "landsat:landsat_scene_id": "LC80880802020146LGN00",
+      "landsat:wrs_path": 88,
+      "landsat:wrs_row": 80,
+      "odc:dataset_version": "3.1.0",
+      "odc:file_format": "GeoTIFF",
+      "created": "2020-06-18T07:07:06.303867Z",
+      "odc:producer": "ga.gov.au",
+      "odc:product_family": "ard",
+      "odc:region_code": "088080",
+      "odc:lineage": [
+          "level1"
+      ],
```

Comment:
Looks like a mistake?

(maybe we need a normalising formatter to make these diffs more useful, every line is changing :slightly_smiling_face:)